Automattic/jetpack
# AI Assistant: Add icon to input and fix loader colors

Fixes #30832
Fixes #30833
Touches on #30716

## Proposed changes:

* Adds the AI Assistant icon to the input
* Fixes the loader color on dark themes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block
* Check that the AI icon is visible to the left of the input
* Check that it is replaced by the loader icon during a request
* Check on different themes

https://github.com/Automattic/jetpack/assets/8486249/29d3e192-e3de-4f57-bc18-38bb5b91fe91
